snap/snapshots.go

```diff
@@ -27,27 +27,79 @@ import (
 	"path/filepath"
 	"strings"
 
-	"gopkg.in/yaml.v2"
-
 	"github.com/snapcore/snapd/strutil"
+	"gopkg.in/yaml.v2"
```

Comment:
why this change? our imports are alwas organzied as:

```
import (
   "stblib"

  "other-libs"

  "snapcore/snapd"
)
```

Reply:
Good point. Fixed